spyder/plugins/workingdirectory/confpage.py

```diff
@@ -62,32 +66,34 @@ def setup_page(self):
         startup_group.setLayout(startup_layout)
 
         # Console Directory
-
-        console_group = QGroupBox(_("Console directory"))
-        console_label = QLabel(_("The working directory for new consoles is:"))
+        console_group = QGroupBox(_("New consoles"))
+        console_label = QLabel(
+            _("The working directory for new IPython consoles is:")
+        )
         console_label.setWordWrap(True)
         console_bg = QButtonGroup(console_group)
         console_project_radio = self.create_radiobutton(
-                                _("The current project directory "
-                                  "or user home directory "
-                                  "(if no project is active)"),
-                                'console/use_project_or_home_directory',
-                                True,
-                                button_group=console_bg)
+            _("The project or user home directory"),
+            'console/use_project_or_home_directory',

```

Comment:
```suggestion
            _("The project (if open) or user home directory"),
            'startup/use_project_or_home_directory',
            tip=_("The working dir for new consoles will be root of the "
                  "project if one is open, otherwise the user home directory"),
```

As above